packages/server/test/question.integration.ts

```diff
@@ -35,23 +35,31 @@ describe('Question Integration', () => {
   describe('GET /questions/:id', () => {
     it('gets a question with the given id', async () => {
       const course = await CourseFactory.create();
+      const student1 = await UserCourseFactory.create({
```

Comment:
out of curiosity what's the reason for adding student1 and manually passing in the question creator?

Reply:
in order to agree with the past snaposhot, i needed the person with ID 1 to submit the question -- but If I create a TA afterwards, then the sender has id 3(unless i specify the sender manually).  I could update the snapshot but this seemed easier to be consistent